x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.test.ts

```diff
@@ -170,6 +188,29 @@ describe('rules_notification_alert_type', () => {
       });
     });
 
+    it('should set a partial failure for when rules cannot read all provided indices', async () => {
```

Comment:
Yeah I'm thinking maybe if the rule doesn't have read access to **any** index we would fail. If it only has read access to a few indices then we would do the partial failure. This is a good test case. I'll update this.